olp-cpp-sdk-core/src/cache/DefaultCacheImpl.cpp

```diff
@@ -817,13 +834,13 @@ DefaultCache::StorageOpenResult DefaultCacheImpl::SetupProtectedCache() {
   auto status = protected_cache_->Open(
       settings_.disk_path_protected.get(), settings_.disk_path_protected.get(),
       protected_storage_settings, OpenOptions::ReadOnly);
-  if (status == OpenResult::Fail) {
+  if (status == OpenResult::Fail || status == OpenResult::Corrupted) {
     OLP_SDK_LOG_ERROR_F(kLogTag, "Failed to reopen protected cache %s",
```

Comment:
"Failed to reopen..."? Is this a copy/paste error? It should be "Failed to open... "